ci-scripts/unit-tests-core-lib.sh

```diff
@@ -1,130 +1,124 @@
 #!/usr/bin/env bash
-set -e
 set -o pipefail
 
+# -azure argument used by azure hyperspace pipeline
```

Comment:
azure pipeline: all tests run even if one fails (as 'set-e' not present). Azure does not support stream target '/dev/fd/5'.
GH pipeline: keep the same behavior as before -> unit test process stops when 1 test fails